notebooks/community/model_garden/model_garden_pytorch_stable_diffusion.ipynb

```diff
@@ -0,0 +1,702 @@
+{
```

Comment:
<p>I assume the training container has the training script and model? Its not obvious.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Added more clarification.